risc0/zkvm/sdk/rust/platform/src/io.rs

```diff
@@ -31,7 +33,11 @@ impl<T> Gpio<T> {
     }
 
     pub const fn as_ptr(&self) -> *mut T {
-        self.addr as _
+        if cfg!(target_os = "zkvm") {
+            self.addr as _
+        } else {
+            unimplemented!()
+        }
```

Comment:
What's this for?

Reply:
Added comment.
            // If we're not on zkvm, don't make it easy to get a                                                                                                                                                   
            // pointer to a GPIO port since it'll point off to the                                                                                                                                                 
            // middle of nowhere.